lib/broccoli/default-packager.js

```diff
@@ -833,30 +834,42 @@ module.exports = class DefaultPackager {
 
         let destDir,
             testSrcTree,
-            srcDir;
+            srcDir,
+            dirNotFound;
 
         // ember-addon
         if (this.name === 'dummy') {
           testSrcTree = 'src';
           destDir = `${this.project.name()}/src`;
+
+          // Classic Addon with MU dummy app `kitchen-sink-with-mu-dummy-app` will throw: `Dir not found: src` exception
```

Comment:
Is there any better alternative than `dirNotFound` to avoid the error? `Funnel.allowEmpty` does not solve the issue.

Reply:
I believe that allowEmpty fails to fix the issue here due to the same underlying issue that is described in https://github.com/broccolijs/broccoli-funnel/issues/105.

I'd like to land https://github.com/broccolijs/broccoli-funnel/pull/106 + release, then we can remove this work around.

What do you think?

Reply:
@rwjblue Of course! I follow the issues and when they will be released, I will get rid of the `dirNotFound` hack.

Reply:
I just released broccoli-funnel@2.0.2 with the fix...

Reply:
@rwjblue Unfortunately `broccoli-funnel@2.0.2` does not fix the issue.

**The error is triggered by `broccoli/lib/builder` before the `BroccoliFunnel.build` is executed; the `BroccoliFunnel.allowEmpty` routine is never executed.**

Any idea what may happen?

Stack trace:

```
  - stack: BuilderError: Directory not found: src
    at Builder.checkInputPathsExist (/Users/ppcano/dev/ember/ember-cli/node_modules/broccoli/lib/builder.js:261:15)
    at new Builder (/Users/ppcano/dev/ember/ember-cli/node_modules/broccoli/lib/builder.js:65:10)
    at Builder.setupBroccoliBuilder (/Users/ppcano/dev/ember/ember-cli/lib/models/builder.js:85:24)
    at new Builder (/Users/ppcano/dev/ember/ember-cli/lib/models/builder.js:32:10)
    at ServeTask.run (/Users/ppcano/dev/ember/ember-cli/lib/tasks/serve.js:45:55)
    at Promise.resolve.then (/Users/ppcano/dev/ember/ember-cli/lib/models/command.js:243:46)
    at tryCatcher (/Users/ppcano/dev/ember/ember-cli/node_modules/rsvp/dist/rsvp.js:323:19)
    at invokeCallback (/Users/ppcano/dev/ember/ember-cli/node_modules/rsvp/dist/rsvp.js:495:31)
    at /Users/ppcano/dev/ember/ember-cli/node_modules/rsvp/dist/rsvp.js:559:14
    at flush (/Users/ppcano/dev/ember/ember-cli/node_modules/rsvp/dist/rsvp.js:2402:5)
```